source/router.ts

```diff
@@ -11,10 +11,11 @@ export class Router extends EventTarget {
 	static parameterNameMatcher = /:[a-zA-Z0-9]+/g;
 	static parameterMatcher = '([^/]+)';
 
-	declare addEventListener: (type: 'parameterchange', callback: EventListenerOrEventListenerObject, options?: boolean | AddEventListenerOptions) => void;
+	declare addEventListener: (type: 'routechange' | 'parameterchange', callback: EventListenerOrEventListenerObject, options?: boolean | AddEventListenerOptions) => void;
 
 	rootNode: Node;
 
+	onRouteChange = () => {};
```

Comment:
rename to lowercase, as with other handlers.

rename onParameterChange too, but keep onRouteChangeEvent with uppercase letters